partition/roles/ztp/files/config_db.json

```diff
@@ -0,0 +1,7 @@
+{
+  "DEVICE_METADATA": {
+    "localhost": {
+      "docker_routing_config_mode": "split-unified"
```

Comment:
TODO: on EC SONiC this is called `split`, for upstream we need `split-unified`. Maybe introduce dynamic config_db.json depending on the SONiC version.

Reply:
Adapted image instead.